examples/with-emotion/pages/_app.js

```diff
@@ -1,19 +1,14 @@
-import NextApp from 'next/app'
+import createCache from '@emotion/cache'
 import { CacheProvider } from '@emotion/core'
+import { globalStyles } from '../shared/styles'
 
-// Use only { cache } from 'emotion'. Don't use { css }.
-import { cache } from 'emotion'
+const cache = createCache()
 
-import { globalStyles } from '../shared/styles'
+const App = ({ Component, pageProps }) => (
+  <CacheProvider value={cache}>
```

Comment:
If the usages of createCache and CacheProvider are removed, the FOUC won't happen

Reply:
Great, thanks for the feedback!

What are the downsides of not using the cache? Would you recommend removing the cache provider also from `_app.js` in this example?

Is it only for customizing the options documented at the link below (eg. prefix, Stylis plugins, key)?

https://emotion.sh/docs/cache-provider

Reply:
No downsides. Yes, it should be removed. Yeah, the options at https://emotion.sh/docs/@emotion/cache#options + https://emotion.sh/docs/ssr#advanced-approach

Reply:
In the interest of keeping things simple I think the with-emotion example code should literally just be the things outlined here:

https://github.com/vercel/next.js/pull/17651#issuecomment-705309908

And exclude the cache and provider, to avoid confusion.